src/caret_analyze/architecture/architecture.py

```diff
@@ -44,97 +47,97 @@ def __init__(
             file_type, file_path)
         loaded = ArchitectureLoaded(reader, ignore_topics)
 
-        self._nodes: Tuple[NodeStructValue, ...] = loaded.nodes
-        self._communications: Tuple[CommunicationStructValue, ...] = loaded.communications
-        self._executors: Tuple[ExecutorStructValue, ...] = loaded.executors
+        self._nodes: Tuple[NodeStruct, ...] = loaded.nodes
+        self._communications: Tuple[CommunicationStruct, ...] = loaded.communications
+        self._executors: Tuple[ExecutorStruct, ...] = loaded.executors
         self._path_manager = NamedPathManager(loaded.paths)
         self._verify(self._nodes)
 
-    def get_node(self, node_name: str) -> NodeStructValue:
+    def get_node(self, node_name: str) -> NodeStruct:
         try:
             return Util.find_one(lambda x: x.node_name == node_name, self.nodes)
```

Comment:
Architectureクラスの出力はNodeStructValueのままでお願いします。
```suggestion
    def get_node(self, node_name: str) -> NodeStructValue:
        try:
            node = Util.find_one(lambda x: x.node_name == node_name, self.nodes)
            return node.to_value()
```
Architectureクラスの内部では変更可能のままにしつつ、出力される型は今までどおりStructValueです。
ほかの関数についても同様にお願いします。